src/typescript/frontend/src/components/trade/OrderEntry/LimitOrderEntry.tsx

```diff
@@ -121,59 +123,59 @@ export const LimitOrderEntry: React.FC<{
                 }
               },
             })}
-            className="h-full w-[100px] flex-1 bg-transparent text-right font-roboto-mono text-xs font-light text-neutral-400 outline-none"
+            className="z-30 w-full bg-transparent pb-3 pl-14 pr-14 pt-3 text-right font-roboto-mono text-xs font-light text-neutral-400 outline-none"
```

Comment:
Makes sure that the input box is above the adornments on the z-axis, so this ensures that you're always clicking on the input element instead of the label or span